BindingContext/BPoP.md

```diff
@@ -76,6 +84,15 @@ One prominent place where authentication cookies may be shared with multiple par
 
 While it is possible for `example.com` to properly audience constrain cookies today (e.g. by issuing one cookie for each subdomain, rather than one cookie in the top-level domain), doing so in practice has proven to be prohibitively cumbersome for many deployments.
 
+### Design proposal
+
+Here is how BPoP is expected to work end-to-end:
+
+1. Webpage user goes to `example.com`, and initiates login. `example.com` redirects to `login.microsoftonline.com`, using javascript or http headers to active binding for `example.com` cookies.
+2. `login.microsoftonline.com`, shows password prompt, javascript or http header activates binding for eSTS cookie
+3. enter password, ESTS SHR in http request, response contains bound eSTS cookie, redirect to example.com with auth code
```

Comment:
A bit confused, who generates SHR in this step?
>>> ESTS SHR in http request

Consider expanding a bit for clarity.

Reply:
Yes, agreed. Sorry, I planned to checkin after expanding this, :). Will get it in the next draft.